wetterdienst/dwd/radar/metadata.py

```diff
@@ -0,0 +1,91 @@
+from enum import Enum
+
+
+class RadarDataType(Enum):
+    """ Enumeration for different Radar Data Types"""
+
+    BUFR = "bufr"
+    HDF5 = "hdf5"
+    BINARY = "binary"
+
+
+class RadarParameter(Enum):
+
+    # Composites
+    # https://docs.wradlib.org/en/stable/notebooks/fileio/wradlib_radar_formats.html#German-Weather-Service:-RADOLAN-(quantitative)-composit  # noqa:E501,B950
+
+    # https://opendata.dwd.de/weather/radar/composit/
+    # TODO: "FX" was missing here. OTOH, "pp" doesn't
+    #       seem to be present on opendata.dwd.de.
+    FX_REFLECTIVITY = "fx"
+    PG_REFLECTIVITY = "pg"
+    # PP_REFLECTIVITY = "pp"
+    WX_REFLECTIVITY = "wx"
+    WN_REFLECTIVITY = "wn"
+    RX_REFLECTIVITY = "rx"
+
+    # https://opendata.dwd.de/weather/radar/radolan/
+    RW_REFLECTIVITY = "rw"
+    RY_REFLECTIVITY = "ry"
+    SF_REFLECTIVITY = "sf"
```

Comment:
@meteoDaniel: I've just added these. Did I give them appropriate names? Are RW, RY and SF also "reflectivity" parameters?

Reply:
No RW: Is a radar based hourly precipitation sum and SF is the same product for 24h sum

Reply:
`RY` is kind of quality corrected reflectivity data (5 minutes interval).

Reply:
@kmuehlbauer  thanks for clarify. Only information available was that RSS-Feed entry: https://www.dwd.de/DE/leistungen/opendata/neuigkeiten/opendata_jul2018_03.html

Reply:
Thanks for your feedback on this.

So, I believe it's just fine to call them `RW_REFLECTIVITY`, `RY_REFLECTIVITY` and `SF_REFLECTIVITY`. Please unresolve conversation if you have any objections.